codegen/post_gen_hooks.go

```diff
@@ -140,9 +144,18 @@ func ClientMockGenHook(h *PackageHelper, t *Template) (PostGenHook, error) {
 					genDir := filepath.Join(buildDir, instance.Directory, "mock-client")
 
 					importPath := importPathMap[key]
+					customInterface := customInterfaceMap[key]
+
+					var mock []byte
+					var err error
+					var clientInterface = "Client"
```

Comment:
defaultInterface is a already a const in the file use that. 

moreover, override it above and use it in pathSymbolMap and create a clientInterfaceMap rather than customInterfaceMap and provide the overriden client value.

Reply:
Done